### DISCUSSION TITLE
Open property for Combobox

### DISCUSSION THREAD

**Discussion by kaiqueaguilar** [chakra-ui/ark#2372] [Ideas]
Combobox has an "onOpenChange" property but does not allow access to the "open" property itself.

Components like Menu, Date Picker, and Popover have an "open" property to control their internal state.

I tried using "present" as an alternative, but it's not in sync with "onOpenChange".

**Comment by segunadebayo** [comment]
Good catch, @kaiqueaguilar. We'll work on adding that to Zag.js shortly

**Reply by segunadebayo** [reply]
Now done.

**Comment by segunadebayo** [comment]
Closing — Combobox now supports `open` and `defaultOpen` props.
